lodash/index.d.ts

```diff
@@ -11202,7 +11202,7 @@ declare namespace _ {
     }
 
     //_.cloneDeepWith
-    type CloneDeepWithCustomizer<TValue, TResult> = (value: TValue) => TResult;
+    type CloneDeepWithCustomizer<TValue, TResult> = (value: TValue, key?: number|string, object?: any, stack?: any) => TResult;
```

Comment:
do not use `?` in callbacks. lodash will always call it with 4 arguments, functions are allowed to ignore arguments.